actors/builtin/power/power_test.go

```diff
@@ -145,6 +148,116 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 		require.Equal(t, big.Zero(), claim2.RawBytePower)
 		require.Equal(t, big.Zero(), claim2.QualityAdjPower)
 	})
+
+	t.Run("power accounting crossing threshold", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+
+		actor.createMinerBasic(rt, owner, owner, miner1)
+		actor.createMinerBasic(rt, owner, owner, miner2)
+		actor.createMinerBasic(rt, owner, owner, miner3)		
+		actor.createMinerBasic(rt, owner, owner, miner4)				
+		actor.createMinerBasic(rt, owner, owner, miner5)
+
+		actor.updateClaimedPower(rt, miner1, smallPowerUnit, mul(smallPowerUnit, 2))
+		actor.updateClaimedPower(rt, miner2, smallPowerUnit, mul(smallPowerUnit, 2))		
+		actor.updateClaimedPower(rt, miner3, smallPowerUnit, mul(smallPowerUnit, 2))				
+
+		actor.updateClaimedPower(rt, miner4, powerUnit, mul(powerUnit, 2))
+		actor.updateClaimedPower(rt, miner5, powerUnit, mul(powerUnit, 2))		
+
+		// Below threshold small miner power is counted
+		expectedTotalBelow := big.Sum(mul(smallPowerUnit, 3), mul(powerUnit, 2))
+		actor.expectTotalPower(rt, expectedTotalBelow, mul(expectedTotalBelow, 2))
+
+		// Above threshold (power.ConsensusMinerMinMiners = 3) small miner power is ignored
+		delta := big.Sub(powerUnit, smallPowerUnit)
+		actor.updateClaimedPower(rt, miner3, delta, mul(delta, 2))
```

Comment:
The QA power is the one that counts, so this puts the power way above the threshold, not just at it. But doing the subtraction implies you meant to _just_ cross the line. The same applies to the miner4/5 set-ups.

It might be clearer to divide for the raw power than multiply for the QA power.